pages/museu/index.public.js

```diff
@@ -88,15 +88,19 @@ export default function Page() {
             <PrimerLink href="/museu/post-02.html">Post POC #02</PrimerLink> feito por{' '}
             <PrimerLink href="https://github.com/pscruzzz">Pedro Cruz</PrimerLink>
           </li>
-        </ul>
-      </Box>
+        </Box>
 
-      <style jsx>{`
-        .list {
-          margin-top: 2rem;
-          list-style: circle;
-        }
-      `}</style>
+        <Text as="p">
+          Abaixo temos um artigo que fala as principais mudanças do TabNews, do início do projeto ao lançamento oficial.
```

Comment:
```suggestion
        <Text as="h2">
          Artigo
```

E se um dia adicionarmos outros, muda o texto para "Artigos".

Eu também levaria esse tópico para o topo da página, por ser algo mais recente e completo. E criaria um outro  `h2` para os primeiros layouts.